Sources/VitaminUIKit/Components/Snackbar/VitaminSnackbar+Configuration.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Vitamin iOS
+//  Apache License 2.0
+//
+
+import UIKit
+
+public extension VitaminSnackbar {
+    /// Struct that can be provided to the initializer of `VitaminSnackBar` or `UIViewController.pushVitaminSncakbar()`
+    /// that groups all basic configuration
+    struct BasicConfiguration {
```

Comment:
Maybe better to have something with semantics like `ContentConfiguration`.

Reply:
ok